CHANGELOG.md

```diff
@@ -156,6 +156,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj new` can now be used for creating merge commits. If you pass more than
   one argument to it, the new commit will have all of them as parents.
 
+* The default revset used by `jj log` can now be configuered using the
+  `ui.default-revset` setting. If not 
```

Comment:
Unfinished sentence. Also, perhaps this should be combined with the item on line 25?

I suppose I should cut a release soon; It's been a long time...